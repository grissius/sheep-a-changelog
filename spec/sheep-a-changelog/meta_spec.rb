require 'sheep-a-changelog/meta'

RSpec.describe 'Meta data' do
  it 'works and is unchanged' do
    meta = %i[author name description email license homepage summary]
           .map { |m| [m, SheepAChangelog::Meta.public_send(m)] }.to_h
    expect(meta).to match_snapshot
  end
end
