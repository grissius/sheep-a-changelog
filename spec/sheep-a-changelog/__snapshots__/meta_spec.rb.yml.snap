---
"./spec/sheep-a-changelog/meta_spec.rb[1:1]":
  :author: Jaroslav Šmolík
  :name: sheep-a-changelog
  :description: Simple, particular, example driven parser of keep-a-changelog format.
  :email: grissius@gmail.com
  :license: MIT
  :homepage: https://github.com/grissius/keep-a-changelog
  :summary: Simple, particular, example driven parser of keep-a-changelog format.
